docs/source/install.rst

```diff
@@ -5,44 +5,70 @@
 
-While the Jupyter Notebook allows users to run code in many different programming
-language, the Notebook itself is implemented in Python.
+While the Jupyter Notebook allows users to run code in many different
+programming language, the Notebook itself is implemented in Python. Thus to
+install the Notebook you will also need Python.
 
-**If you already have Python**, you can install the notebook using pip::
-
-    pip install jupyter
 
+If you are new to Python
+........................
 
-**If you're new to Python**, we recommend installing `Anaconda
-<http://continuum.io/downloads#py34>`_, a Python distribution which includes
-Jupyter. After you install it, ensure that Jupyter is up to date by running::
+If you're new to Python we recommend installing `Anaconda
+<http://continuum.io/downloads#py34>`_, which includes Python itself, the Jupyter
+Notebook and other packges for scientific computing and data science. After you
+install it, ensure that Jupyter is up to date by running::
 
     conda update jupyter
 
+If you already have Python
+..........................
+
+If you already have Python, you can install the Jupyter Notebook using pip::
+
+    pip install jupyter
+
 .. note::
 
-   Jupyter has just been released, and it will be packaged for conda in a few
-   days. In the meantime, all of the Jupyter functionality is included in
-   the latest packaged version of IPython. Make sure that the ``ipython`` and
-   ``ipython-notebook`` packages are installed.
+     The above commands should be run Terminal (Mac/Linux) or Command Prompt
+     (Windows)
+
+Upgrading
+.........
+
+The Jupyter Notebook used to be called the IPython Notebook. If you are running
+an older version of the IPython Notebook (3 or earlier) you can use the
+following to upgrade to the latest version of the Jupyter Notebook.
 
-To **upgrade from IPython 3 or older to Jupyter**, use pip or conda—according
-to which one IPython was installed with—to install the ``jupyter`` package. This
-should install the Jupyter Notebook and upgrade IPython to version 4.
+If the older version of the IPython Notebook was installed using `pip`::
+
+    pip install -U jupyter
+    
+If the older version of the IPython Notebook was installed using `conda`::
+
+    conda update jupyter
 
 Installing kernels
 ------------------
 
-Installing the Jupyter Notebook should also have installed the `IPython kernel
-<http://ipython.readthedocs.org/en/master/>`_ for the Python programming language.
+Installing the Jupyter Notebook as described above will also install the `IPython
+kernel <http://ipython.readthedocs.org/en/master/>`_ for working in the Python programming language.
+
 
 To run notebooks in other languages, you'll need to install additional kernels:
 see the `list of available kernels
 <https://github.com/ipython/ipython/wiki/IPython-kernels-for-other-languages>`_.
+
+Next steps
+----------
+
+After installing you are ready to :ref:`run the notebook <running>`.
```

Comment:
is "run" the term we want to use ? (here and in next section). 

You might want to also precise "Notebook Server" or it ass confusion document/app, let's try to be (even implicitely) precise when refering to Server/Client-side app/ document.